test/functional/wallet.py

```diff
@@ -19,6 +19,7 @@ def __init__(self):
         self.setup_clean_chain = True
         self.num_nodes = 4
         self.extra_args = [['-usehd={:d}'.format(i%2==0)] for i in range(4)]
+        self.extra_args[0].append('-txindex')
```

Comment:
@jnewbery this is why I prefer isolated tests. This is unrelated to all these tests except the new.